src/app/modules/item/components/item-children-edit/item-children-edit.component.ts

```diff
@@ -60,6 +61,11 @@ export class ItemChildrenEditComponent implements OnChanges {
     this.data.push({ title: child.title, type: child.type });
   }
 
+  deleteSelected(): void {
```

Comment:
Call it "remove", that's is more correct than "delete". And "*Selected" is not very clear either.
So call it, for instance, `onRemove`

Reply:
Done